content/about/main/index.md

```diff
@@ -7,29 +7,29 @@ number_categories: 3 # set to zero to exclude
 show_intro: true
 intro: |
 
-  I was born in 1977 in [Juan José Paso](https://es.wikipedia.org/wiki/Juan_Jos%C3%A9_Paso_(Buenos_Aires)), a small town in [Argentina](https://es.wikipedia.org/wiki/Argentina). I was six years old when my elementary school bought a computer (one computer for the entire school and the only one in the whole town). The moment I saw it I felt love at first sight.
+  I was born in 1977 in [Juan José Paso](https://es.wikipedia.org/wiki/Juan_Jos%C3%A9_Paso_(Buenos_Aires)), a small town in [Argentina](https://es.wikipedia.org/wiki/Argentina). I was six year old when my elementary school bought a computer (one computer for the entire school and the only one in the whole town). The moment I saw it I felt love at first sight.
   
   A few years later, we moved to [Ingeniero Luiggi](https://es.wikipedia.org/wiki/Ingeniero_Luiggi) because my family had lost almost everything due to a flood. This new place was close to [Realicó](https://es.wikipedia.org/wiki/Realic%C3%B3) where I could study programming. So between ages 9 and 14 my parents drove 80 kilometers every Saturday to take me to class.  We didn’t have a car, and there was no public transport because it was a rural area, but since my dad repairs vehicles, his clients would lend him their cars to make the trip.
   
-  When I was 15, I started working teaching informatics. Since then, I never stopped teaching and learning computing. In 1996 I become the first generation in my family in obtein a high school diploma. At that time the whole family moved again, to [Santa Rosa](https://es.wikipedia.org/wiki/Santa_Rosa_(La_Pampa)) with the hope I can study at the university. My family come with me because we didn't have the money to send me to study.  I couldn't stop working, so I was a teacher at elementary and high school and I teach to teacher through the _Red Federal de Formación Docente Continua (Ministry of Education)_ from 1996 to 1999. I also develop software and made my first sold (a car racing management system) in 1997.
+  When I was 15, I started working teaching informatics. Since then, I have never stopped teaching and learning computing. In 1996 I became the first generation in my family obtaining a high school diploma. At that time the whole family moved again, to [Santa Rosa](https://es.wikipedia.org/wiki/Santa_Rosa_(La_Pampa)) with the hope I can study at the university. My family came with me because we didn't have the money to send me to study.  I couldn't stop working, so I was a teacher at elementary and high school and I taught to teachers through the _Red Federal de Formación Docente Continua (Ministry of Education)_ from 1996 to 1999. I also starting developing software and made my first paid project (a car racing management system) in 1997.
```

Comment:
is "paid project" what you mean?

Also I suspect other nerds might like to know what tools/languages were used!

Reply:
By the way when did you first use R? That's a very important piece of information / fun fact missing from here :stuck_out_tongue_winking_eye:

Reply:
paid project means I sold the software :-). I use FoxPro.  I use R for the first time on 2009, in one of the course of my master. I will add this information!!